config.py

```diff
@@ -27,7 +27,6 @@ class Default(object):
     MAIL_DEFAULT_SENDER = os.environ.get("MAIL_DEFAULT_SENDER")
     MAIL_MAX_EMAILS = os.environ.get("MAIL_MAX_EMAILS", 3)
     MAIL_ASCII_ATTACHMENTS = os.environ.get("MAIL_ASCII_ATTACHMENTS", False)
-    MAIL_SUPPRESS_SEND = os.environ.get("MAIL_SUPPRESS_SEND", False)
```

Comment:
Can you explain this removal for me?

Reply:
`MAIL_SUPRESS_SEND` is not a configuration option in [Flask-Mailman.](https://www.waynerv.com/flask-mailman/). And neither are `MAIL_MAX_EMAILS` and `MAIL_ASCII_ATTCHMENTS`. There's no harm in leaving unused configurations in the application, but I think it can cause confusion.

Reply:
Yep, agreed. Let's remove all unneeded configs from all **three** dev environments then. 

It looks like Flask-Mailman suppresses mails in the testing environment without the use of configs, which is great. It probably be great to have a tool that catches mail in the dev environment and allows you to view them. But I'm not sure how involved that would be and it sounds like a different issue then this one. I think the Flask-Mailman docs mentioned something about it, but I'm not sure.

Reply:
I've removed the unused configuration options, and will look into seeing if we can set something up in the dev environment to actually review the emails sent.

Reply:
Using `MAIL_BACKEND = 'console'` will display the sent email to the console. I've tested it manually, and it works.

Reply:
That works! Thank you 😄